docs/attributes-registry/http.md

```diff
@@ -13,8 +13,10 @@
 | `http.request.method` | string | ![Stable](https://img.shields.io/badge/-stable-lightgreen)<br>HTTP request method. [2] | `GET`; `POST`; `HEAD` |
 | `http.request.method_original` | string | ![Stable](https://img.shields.io/badge/-stable-lightgreen)<br>Original HTTP method sent by the client in the request line. | `GeT`; `ACL`; `foo` |
 | `http.request.resend_count` | int | ![Stable](https://img.shields.io/badge/-stable-lightgreen)<br>The ordinal number of request resending attempt (for any reason, including redirects). [3] | `3` |
+| `http.request.size` | int | The total size of the request in bytes. This should be the total number of bytes sent over the wire, and includes all framing and control data, headers, and request body if any. For requests using transport encoding, this should include the compressed body size as sent over the wire. | `1437` |
 | `http.response.body.size` | int | The size of the response payload body in bytes. This is the number of bytes transferred excluding headers and is often, but not always, present as the [Content-Length](https://www.rfc-editor.org/rfc/rfc9110.html#field.content-length) header. For requests using transport encoding, this should be the compressed size. | `3495` |
 | `http.response.header.<key>` | string[] | ![Stable](https://img.shields.io/badge/-stable-lightgreen)<br>HTTP response headers, `<key>` being the normalized HTTP Header name (lowercase), the value being the header values. [4] | `http.response.header.content-type=["application/json"]`; `http.response.header.my-custom-header=["abc", "def"]` |
+| `http.response.size` | int | The total size of the response in bytes. This should be the total number of bytes sent over the wire, and includes all framing and control data, headers, and response body and trailers if any. For responses using transport encoding, this should include the compressed body size as sent over the wire. | `1437` |
```

Comment:
this differs from the yaml file. Which one is correct then?
The source of truth should be yaml file and md files are generated from them using `make table-generation`